### PR TITLE
[Utopia-402] new home page

### PR DESCRIPTION
**Pull request by binggaofw**
# Description

This PR includes the following proposed change(s):
- update ppq home page to latest design
Screen Shot 2022-11-14 at 4 37 02 PM

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Refactoring / Documentation
- [ ] Version change

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes.

## Development Dependency Working Agreement
- [x] My code DOES NOT include the importing of new dependencies into the DPIA ecosystem
- [ ] My code DOES include the importing of new dependencies into the DPIA ecosystem
**If new dependencies are being introduced to the DPIA ecosystem:**
- [ ] The functionality of the dependency drastically reduces code complexity and makes my changes more easily maintainable and readible 
- [ ] The dependency being introduced does not contain multiple layers of nested dependencies introducing maintainability complexity to the DPIA ecosystem

## Frontend Development Changes
- [ ] N/A
- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to project documentation or diagrams that reflect my changes
- [ ] New and existing unit tests pass locally with my changes
- [ ] My code follows Airbnb React Style Guidelines

## API Development Changes
- [x] N/A
- [ ] I have performed a self-review of my own code
- [ ] My code follows standards and practices outlined in the BC Government API Development Guidelines
- [ ] New and existing unit tests pass locally with my changes
- [ ] My changes includes Swagger documentation updates that reflect the changes I am introducing

## Definition of Done

Definition of Done

### CODE REVIEW

**Comment by Bottle7** [comment]
The new component needs padding and margin. Also, the blue needs to be darker (is that the color code from the design?). We should also make the button background transparent to allow for the background color of the component to show through as per the design.

**Comment by binggaofw** [comment]
@Bottle7 yes, the color i got from Figma and convert to rgba format from hex

File: src/frontend/src/pages/PPQPage/PPQPage.tsx [hunk 1]
```diff
@@ -11,20 +14,20 @@ function PPQLandingPage() {
       <div className="ppq-section results-wrapper">
         <div className="row what-is-a-ppq">
           <div className="col-md-6">
-            <h2>PIA Pathway Questionnaire</h2>
+            <h2>PIA Intake</h2>
```
Comment (Bottle7) [inline src/frontend/src/pages/PPQPage/PPQPage.tsx:17]: This heading needs a `font-weight: 700`

**Comment by Bottle7** [comment]
We still need padding inside of the new component.

image

File: src/frontend/src/pages/PPQPage/PPQPage.tsx (hunk 1 above)
Comment (binggaofw) [reply src/frontend/src/pages/PPQPage/PPQPage.tsx:17]: i change it to h1, is that ok?

**Comment by binggaofw** [comment]
> We still need padding inside of the new component.
> 
> image

I add padding to the ppq div element, please let me know if it is ok @Bottle7

File: src/frontend/src/pages/PPQPage/PPQPage.scss [hunk 2]
```diff
@@ -14,7 +14,18 @@
...
+
+.ppq-content {
+  padding: 25px;
+  box-sizing: border-box;
+  margin-right: 4em;
+  background: rgba(59, 153, 252, 0.05);
+}
+
+.ppq-home-warning-icon {
+  color: #fcba19;
```
Comment (antsand) [inline src/frontend/src/pages/PPQPage/PPQPage.scss:28]: Can we create a color variable?  This will start to create inconsistencies in UI/UX elements.

**Comment by Bottle7** [comment]
@binggaofw The padding better now; however, the PPQLandingPage layout is broken now.

File: src/frontend/src/pages/PPQPage/PPQPage.scss (hunk 2 above)
Comment (binggaofw) [reply src/frontend/src/pages/PPQPage/PPQPage.scss:28]: fix it

File: src/frontend/src/pages/PPQPage/PPQPage.tsx (hunk 1 above)
Comment (antsand) [reply src/frontend/src/pages/PPQPage/PPQPage.tsx]: Can we have a broader discussion on this. From my understanding we should set the style for H1, H2, p etc and never have to touch font styling ever again. Can we spend some time on defining this at the root level?
https://developer.mozilla.org/en-US/docs/Learn/Accessibility/HTML#good_semantics

Below are detailed links on how we should go about doing this.  

https://developer.mozilla.org/en-US/docs/Learn/Accessibility/HTML
https://developer.mozilla.org/en-US/docs/Learn/Accessibility/CSS_and_JavaScript

File: src/frontend/src/pages/PPQPage/PPQPage.scss [hunk 3]
```diff
@@ -14,7 +14,18 @@
...
   padding: 0;
-  padding-left: 10rem;
+  padding-left: 5rem;
+}
+
+.ppq-content {
+  padding: 25px;
+  box-sizing: border-box;
+  margin-right: 4em;
+  background: rgba(59, 153, 252, 0.05);
```
[the remark this answers is not in the document]
Comment (binggaofw) [reply src/frontend/src/pages/PPQPage/PPQPage.scss]: add a variable to _variables.scss

**Comment by binggaofw** [comment]
close this pr for now as we are implementing feature flaging

**Pull request closed without merging**